lib/components/artist/related_artists/index.js

```diff
@@ -10,6 +10,8 @@ import SerifText from '../../text/serif';
 import ImageView from '../../opaque_image_view';
 import RelatedArtist from './related_artist'
 
+const window = Dimensions.get('window')
```

Comment:
Noice! I wonder how that works under the hood, will have to look into that some time.